Parsers/!lmgtfy or !google search term.js

```diff
@@ -0,0 +1,35 @@
+/*
+activation_example:!lmgtfy or !google search term 
+regex:(SN)||(HCLTECH)||(SERVICENOW)
+flags:gmi
+*/
+
+var where = 0;
+var which = '';
+
+if (current.text.indexOf('!lmgtfy') > -1) {
+
+    where = current.text.indexOf('!lmgtfy') + 8;
+    which = 'lmgtfy';
+
+
+} else if (current.text.indexOf('!google') > -1) {
+    where = current.text.indexOf('!google') + 8;
+    which = 'google';
+}
+
+var term = current.text.substr(where).trim();
+
+if (term == '') {
+    var send_confusion = new x_snc_slackerbot.Slacker().send_chat(current.channel, ':upside_down_face: !docs *something*', false, '', current.thread_ts);
```

Comment:
I don't think you should be referencing `!docs` here

Reply:
Yes it is referencing if term is empty

Reply:
if the term was empty you wouldn't want the user to type `!docs ${their query}`. You want them to type `!lmgtfy` or `!google` so your message should reflect that, probably with something like:
```javascript
':upside_down_face: !' + where + ' *something*'
```